__tests__/pet.test.js/pet.test.js

```diff
@@ -0,0 +1,221 @@
+const Pet = require('../../src/pet');
+
+describe('constructor', () => {
+    it("returns an object", () => {
+        expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+
+
+});
+
+describe('constructor', () => {
+    it('sets the name property', () => {
+        const pet = new Pet('Fido');
+
+        expect(pet.name).toEqual('Fido');
+    });
+});
+
+describe('constructor', () => {
+   it('has a initial age of 0', () => {
+    const pet = new Pet('Fido');
+    
+    expect(pet.age).toEqual(0);
+   });
+    
+});
+
+describe('growUp', () => {
+    it('increments the age by 1', () => {
+        const pet = new Pet('Fido');
+        pet.growUp();
+        expect(pet.age).toEqual(1);
+    });
+
+    it('throws an error if the pet is not alive', () => {
+        const pet = new Pet('Fido');
+  
+        pet.age = 30;
+  
+        expect(() => pet.growUp()).toThrow('Your pet is no longer alive :(');
+      });
+});
+
+describe('gettingUnhealthier', () => {
```

Comment:
You dont have to write the test names using a given coding naming convention, it can be normal language. Something like 'Side effects of aging' would be fine here